### DISCUSSION TITLE
Swiper onAutoplayTimeLeft

### DISCUSSION THREAD

**Discussion by kennsorr** [nolimits4web/swiper#6697] [General]
Hi, I recently saw a demo of `onAutoplayTimeLeft` usage [here](https://codesandbox.io/p/sandbox/33hyfn?file=%2Fsrc%2FApp.jsx%3A19%2C25). On CodeSandBox, it works, but I installed Swiper with npm, and it looks like it's not included. Is there something I'm missing?

https://codesandbox.io/p/sandbox/33hyfn?file=%2Fsrc%2FApp.jsx%3A19%2C25

![image](https://github.com/nolimits4web/swiper/assets/16807664/04e57037-9f92-40cb-a9b9-603fdcc0f066)

**Comment by frankgin** [comment]
@kennsorr 

Need to upgrade your Swiper package. the property `timeLeft` had been updated since version 9.0. https://unpkg.com/browse/swiper@9.0.0/types/modules/autoplay.d.ts
